### PR TITLE
Restored hotkey functionality

### LINKED ISSUE #3: #29 / Add hotkeys
Refheap offered some Hotkeys, which can also be seen in the help bubble. Those hotkeys should return.

- [x] Setup keybindings with placeholder functions

When creating a paste:
- [x] CTRL+Enter => Create Paste
- [ ] ALT+W => Toggle Word Wrap

When viewing a paste:
 - [ ] ALT+G => Go to a line
 - [ ] ALT+CTRL+E => Edit the paste
 - [ ] ALT+R => Show raw code

URL options (when creating):
 - [ ] ?lang=Javascript to default to JS
 
URL options (when viewing):
 - [ ] #L-N Jump to line number N

### PR DESCRIPTION
**Pull request by ghost**
To restore the hotkeys I brought in mousetrap and moustrap
global. index.html now pulls in these scripts. In the process of
adding mousetrap I moved all of the script tags in index.html to just
before the close of body as this has become an industry best practice.

In refheap.js and create.js I exposed the codemirror editor and the
refheap module on the window object. This was necessary to keep all of
the hotkey binding in one place.

resolves macskay/pasteit#3

### CODE REVIEW

File: pasteit/views/index.html [hunk 1]
```diff
@@ -135,5 +129,18 @@
 })();
 //]]>
         </script>
+
```
Comment (macskay) [inline pasteit/views/index.html]: Why did you pull the scripts down to the end of the file?

**Comment by macskay** [comment]
Hmm.. why is it that there are two input textareas?

image

Also when submitting a paste its not shown on the next side.

**Review by macskay — DISMISSED** [review]
See above

**Comment by ghost** [comment]
CodeMirror.fromTextArea was being called twice. This was introduced when editor was added to the window object. This has been fixed.

**Comment by macskay** [comment]
grafik

Using these as a reference I get the following results:
 - [ ] ALT + P (obsolete, since no private/public modes) => should be deleted from the list
 - [x] CTRL + Enter (working as expected)
 - [ ] ALT + W (not working on windows, nothing is happening)
 - [ ] ALT + G (box only opening in insert mode but nothing happens, not in viewing mode)
 - [ ] ALT + CTRL + E (not working on windows, nothing is happening)
 - [ ] ALT + R (nothing is happening)
 - [ ] ?lang=Javascript (not working, no language is selected)
 - [ ] #L-N Jump to line (not working)

**Comment by macskay** [comment]
For documentation purposes: We decided to get rid off the hotkeys.

**Pull request closed without merging**